packages/UI/src/components/molecules/Alert/index.ts

```diff
@@ -1,4 +1,5 @@
 export interface AlertProps {
   signal?: 'notify' | 'success' | 'error';
+  handleClose?: () => void;
```

Comment:
the type here is not so accurate `handleClose?: (e?: React.MouseEvent<HTMLButtonElement, MouseEvent>) => void;` is more explicit

Reply:
Thanks for your feedback, Mohammed! I'll make these changes soon as possible

Reply:
@helioLJ I made a PR to fix those and also add animation to the alert https://github.com/dev-launchers/dev-launchers-platform/pull/1253